app/src/main/java/org/oppia/android/app/home/promotedlist/ComingSoonTopicsViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.android.app.home.promotedlist
+
+import android.graphics.Color
+import androidx.annotation.ColorInt
+import org.oppia.android.app.home.HomeItemViewModel
+import org.oppia.android.app.home.topiclist.DARKEN_SATURATION_MULTIPLIER
+import org.oppia.android.app.home.topiclist.DARKEN_VALUE_MULTIPLIER
+import org.oppia.android.app.home.topiclist.TopicSummaryClickListener
+import org.oppia.android.app.model.TopicSummary
+import org.oppia.android.app.model.UpcomingTopic
+import java.util.*
```

Comment:
Avoid wildcard imports.

Reply:
Removed this import statement as it was unused.